superset/stats_logger.py

```diff
@@ -29,6 +29,9 @@ def decr(self, key):
         """Decrement a counter"""
         raise NotImplementedError()
 
+    def timing(self, key, time_in_milliseconds):
```

Comment:
If I'm reading [this](https://github.com/jsocol/pystatsd/blob/fadc6ccc768c327c830044dee34dd0271302e8d4/statsd/client/base.py#L26) correctly the value can be either ms or a `timedelta` object, thus I wonder if `time_in_milliseconds` should simply be `delta` or `value` (for consistency).